.travis.yml

```diff
@@ -23,6 +23,8 @@ install:
   - git clone https://github.com/OCA/bank-payment.git ${HOME}/bank-payment -b ${VERSION}
   - git clone https://github.com/OCA/partner-contact.git ${HOME}/partner-contact
   - git clone https://github.com/OCA/account-fiscal-rule.git ${HOME}/account-fiscal-rule
+  - git clone https://github.com/OCA/reporting-engines.git ${HOME}/reporting-engines
+  - git clone https://github.com/OCA/account-financial-reporting.git ${HOME}/account-financial-reporting
```

Comment:
I don't understand why runbot is failing
> except_orm: ('Error', u"You try to install module 'l10n_fr_mis_reports' that depends on module 'mis_builder'.\nBut the latter module is not available in your system.").

Maybe use oca_dependencies will fix the bug.